src/Microsoft.Health.Fhir.Api/Features/Audit/AuditHelper.cs

```diff
@@ -109,11 +110,10 @@ private void Log(AuditAction auditAction, HttpStatusCode? statusCode, HttpContex
             if (!string.IsNullOrEmpty(auditEventType) && !FhirAnonymousOperationTypeList.Contains(auditEventType, StringComparer.OrdinalIgnoreCase))
             {
                 // Note: string.Replace() is to suffice a code scanning alert for log injection.
-                var operationType = httpContext.Request?.Method;
-                var sanitizedOperationType = UnknownOperationType;
-                if (!string.IsNullOrWhiteSpace(operationType) && !ValidOperationTypes.Contains(operationType))
```

Comment:
This previous attempt should've  had the not operator ! removed so that it checks that it does contain it.
if (!string.IsNullOrWhiteSpace(operationType) && ValidOperationTypes.Contains(operationType))

Reply:
Fixed it in my last commit.